src/App.css

```diff
@@ -123,43 +282,113 @@ ul {
 .proBtn {
   width: 11.5rem;
   height: 2.063rem;
-  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  margin: 0.75rem 0 0.563rem 3.688rem;
   border-radius: 3px;
   background-color: #0290ff;
+  font-family: RobotoSlab, Arial, sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: white;
+  margin-left: 3.8rem;
+  margin-top: 2rem;
+  border: none;
+}
+
+.MainInp {
+  margin-left: 8.8rem;
 }
 
 .inputContainer {
   display: flex;
   flex-direction: row;
   column-gap: 18%;
-  margin-left: 3%;
 }
```

Comment:
- [ ] Good job in implementing the changes requested by the previous reviewer, however, according to the previous reviewer, you should ensure that your design matches the given requirement. as it is right now the contents of your `.inputContainer` class is overlapping each other, kindly align the margin properties of it contents.  Matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustration of yours and the given Design, please pay attention on the red markings on your design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student](https://github.com/faisalakb/BookStore/assets/43528594/a82dc64e-f5da-469a-9f2e-e2f46bd9fadd)


</td>
      <td>

![studesign](https://github.com/faisalakb/BookStore/assets/43528594/9d3212c0-81bb-41b7-82df-86612b89f5f6)

</td>
    </tr>
  </table>